pkg/plugins/manager/loader/loader.go

```diff
@@ -192,19 +192,22 @@ func (l *Loader) loadPlugins(ctx context.Context, src plugins.PluginSource, foun
 			var err error
 
 			cctx, canc := context.WithTimeout(ctx, time.Second*10)
-			p.AngularDetected, err = l.angularInspector.Inspect(cctx, p)
+			p.AngularMeta.AngularDetected, err = l.angularInspector.Inspect(cctx, p)
 			canc()
 
 			if err != nil {
 				l.log.Warn("Could not inspect plugin for angular", "pluginID", p.ID, "err", err)
 			}
 
 			// Do not initialize plugins if they're using Angular and Angular support is disabled
-			if p.AngularDetected && !l.cfg.AngularSupportEnabled {
+			if p.AngularMeta.AngularDetected && !l.cfg.AngularSupportEnabled {
 				l.log.Error("Refusing to initialize plugin because it's using Angular, which has been disabled", "pluginID", p.ID)
 				continue
 			}
 		}
+		if settings, ok := l.cfg.PluginSettings[p.ID]; ok {
+			p.AngularMeta.DisableDeprecationUIFeatures = settings["disable_angular_deprecation_ui"] == "true"
```

Comment:
I'm not sure if this is the best way to do this (accessing l.cfg.PluginSettings map directly from the loader). By taking a quick look I couldn't find a better way but maybe I am missing something

Reply:
I guess this is fine, I don't see an issue with it (only the naming).

Reply:
here it would be `disable_angular_warnings`